### DISCUSSION TITLE
Feature in old lib (1.4) not in new subgraph (graph in graph)

### DISCUSSION THREAD

**Discussion by syluna** [eckig/graph-editor#196] [Q&A] [answered]
Hello, 
I was using this feature in version 1.4 (Java 8) of the library. 
I am migrating to J17 and taking the opportunity to upgrade to version 19. I couldn't find what this feature has been replaced with and how I could adapt without reusing the code from the old library. I imagine there is a reason, but I don't know enough about EMF to be relevant. 
Thank you for your feedback.

**Comment by eckig** [comment]
Yes, I removed it, as it was never fully implemented / supported.

Do you need this feature back?

**Comment by syluna** [comment]
I don't want to do this just for me. I get the code from the 1.4 and build extlib to add only subgraph (supergraph is not required for me). 

I use subgraph to add "text information" in a parent node and render it in the graph.

**Reply by eckig** [reply, marked as answer]
Normally additional information (like text) is provided with a [domain model](https://github.com/eckig/graph-editor/wiki/4.-Connecting-to-Domain-Models).

**Reply by syluna** [reply]
Yes Ok I try to migrate with this.
